CHANGELOG.md

```diff
@@ -1,5 +1,36 @@
+
+### Release notes 
+* Behavioral changes
+  * If your VM SKU does not support ephemeral or PremiumSSD OS disks, AKS will now use StandardSSD as the default OS disk type. Before, only VM SKUs that did not support ephemeral OS disks had this default.
+
+* Features
+  * Configurable resource group for the Private Link Service (PLS) creation using the *"ServiceAnnotationPLSResourceGroup = "service.beta.kubernetes.io/azure-pls-resource-group"* annotation. [Upstream link](https://github.com/kubernetes-sigs/cloud-provider-azure/pull/4675).
+  * Support customizing probe config when *externalTrafficPolicy* = *local* and *health_probe_port* annotation is defined for service port. [upstream link](https://github.com/kubernetes-sigs/cloud-provider-azure/pull/4639)
+
+* Preview Features
+  * Load Balancer TCP reset can now be disabled with a service annotaiton *"service.beta.kubernetes.io/azure-load-balancer-disable-tcp-reset=true"* in preview. [Upstream link](https://github.com/kubernetes-sigs/cloud-provider-azure/pull/4520)
+  * [Image Integrity](https://learn.microsoft.com/azure/aks/image-integrity) allows you to sign container images via a process that ensures their authenticity and integrity.
+    
+* Bug Fixes 
+  * Fix for the possibility of duplicate podCIDR allocation when node PodCIDR patch request fails. [Upstream link](https://github.com/kubernetes-sigs/cloud-provider-azure/pull/4568)
+  * Fix for the Private Link Service (PLS) creation failure that can occur if the customer selects a subnet name or PLS name that is too long. [Upstream Link](https://github.com/kubernetes-sigs/cloud-provider-azure/pull/4538)
+
+* Component Updates
+  * Security Publisher image version has been updated to 1.0.67 from 1.0.64 which includes memory utilizaiton improvements and increased input buffer size in the fluent-bit configuration.
+  * Cilium version has been updated to [1.13.5](https://github.com/cilium/cilium/releases/tag/v1.13.7) for AKS clusters with kubernetes versions 1.28 or greater
+  * Azure File CSI driver updated to version [v1.24.9](https://github.com/kubernetes-sigs/azurefile-csi-driver/releases/tag/v1.24.9) for clusters with kubernetes version [1.25]"(https://github.com/kubernetes-sigs/azurefile-csi-driver/releases/tag/v1.26.7), v1.26.7 for clusters with kubernetes version 1.26 and [v.1.28.4](https://github.com/kubernetes-sigs/azurefile-csi-driver/releases/tag/v1.28.4) for clusters with kubernetes version 1.27
+  * Hotfix: There were 3 CVE's in the upstream Kubernetes related to insufficient input sanitiztion which leads to privilege escalation. AKS Patched the AKS cluster nodes for clusters version 1.24.9, 1.24.10, 1.24.15, 1.25.5, 1.25.6, 1.25.11, 1.26.0, 1.26.3, 1.26.6, 1.27.3. CVE links - [CVE-2023-3676](https://github.com/Azure/AKS/issues/3869), [CVE-2023-3955](https://github.com/Azure/AKS/issues/3870), and [CVE-2023-3893](https://github.com/Azure/AKS/issues/3871)
+
```

Comment:
Would it be wise to have a call to action for customers to update their node images here?

Reply:
Thanks @PixelRobots for the feedback.  The links themselves have the calls to action for customers.

Reply:
I would include a sentence like:

If your cluster doesn't have automatic node image updates, [update your node image](https://learn.microsoft.com/en-us/azure/aks/node-image-upgrade) to ensure you get the security patch.